src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -100,7 +101,12 @@ class LicenceConditionService(
 
     auditService.recordAuditEventAddAdditionalConditionOfSameType(licenceEntity, newCondition)
 
-    return transform(newCondition)
+    val readyToSubmit = checkConditionsReadyToSubmit(
+      listOf(newCondition),
+      licencePolicyService.policyByVersion(licenceEntity.version!!).additionalConditions,
+    )[newCondition.conditionCode]!!
```

Comment:
could just add a helper for dealing with a single condition in `checkConditionsReadyToSubmit `